includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -492,7 +492,9 @@ protected function get_posted_field( $key, $field ) {
 		if ( ! isset( $field['sanitizer'] ) ) {
 			$field['sanitizer'] = null;
 		}
-		return isset( $_POST[ $key ] ) ? $this->sanitize_posted_field( $_POST[ $key ], $field['sanitizer'] ) : '';
+
+		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized
```

Comment:
I guess the violation here is because it's being run through a custom sanitization method, and not one of the usual recognised functions? If so, add a reason to the ignore to explain as much, to make it more obvious:

```
// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- $this->sanitize_posted_field() checks for foo and then calls sanitize_text_field()...
```

(for example).